src/test/java/com/uid2/shared/store/SiteTest.java

```diff
@@ -1,23 +1,39 @@
 package com.uid2.shared.store;
 
+import com.uid2.shared.model.ClientType;
 import com.uid2.shared.model.Site;
 import org.junit.jupiter.api.Test;
 
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashSet;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class SiteTest {
     @Test
     public void testEquals() {
-        Site a = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), true, 0L);
-        Site b = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), true, 0L);
-        Site c = new Site(1, "1 name", "1 desc", true, new HashSet<>(), new HashSet<>(), null, 0L);
+        Site a = new Site(1, "1 name", "1 desc", true, new HashSet<>(Collections.singletonList(ClientType.DSP)), new HashSet<>(Collections.singletonList("test.com")), new HashSet<>(Arrays.asList("123456789", "com.123.Game.app.Android")), true, 0L);
+        Site b = new Site(1, "1 name", "1 desc", true, new HashSet<>(Collections.singletonList(ClientType.DSP)), new HashSet<>(Collections.singletonList("test.com")), new HashSet<>(Arrays.asList("123456789", "com.123.Game.app.Android")), true, 0L);
```

Comment:
minor: would be good to have a `Site b2` which swap "123456789" and "com.123.Game.app.Android" around and app test that the change of array ordering shouldn't matter for equals method
`assertTrue(a.equals(b2));`